__tests__/app.test.js

```diff
@@ -0,0 +1,40 @@
+const request = require('supertest')
+const app = require('../app')
+const data = require('../db/data/test-data')
+const seed = require('../db/seeds/seed')
+const db = require('../db/connection')
+
+beforeEach(() => {
+    return seed(data)
+})
+
+afterAll(() => {
+    return db.end()
+})
+
+describe('GET /api/topics', () => {
+    test('200: returns array of all topics', () => {
+        return request(app)
+        .get("/api/topics")
+        .expect(400)
+        .then(({ body }) => {
+            const { topics } = body
+            expect(Array.isArray(topics)).toBe(true)
```

Comment:
Since we are using a forEach on line 24, it is implicit that this is an array so we do not need to test for that 🙂